tests/test_browse.py

```diff
@@ -18,6 +19,17 @@ def setUp(self):
         app.config['APPLICATION_ROOT'] = ''
         self.app = app.test_client()
 
+    def test_home(self):
+        rv = self.app.get('/')
+        self.assertEqual(rv.status_code, 200)
+        html = BeautifulSoup(rv.data.decode('utf-8'), 'html.parser')
+
+        for group_key, group_value in taxonomy.GROUPS.items():
+            if group_key == 'grp_test':
+                continue
+            auths_elmt = html.find('h2', text=group_value['name'])
+            self.assertTrue(auths_elmt, f"{group_value['name']} in h2 element")
+
```

Comment:
In that continue on line 29, maybe add a test to make sure there are no groups named 'test'?